spring-integration-core/src/main/java/org/springframework/integration/store/SimpleMessageStore.java

```diff
@@ -58,6 +58,12 @@
 	private volatile boolean isUsed;
 
 	private volatile boolean copyOnGet = false;
+	
+	private final int maxWaitInMilliseconds;
```

Comment:
I named it based off what variable name would make the most sense in the javadocs. It should absolute have the Milliseconds in the variable name imo to denote the units.